packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js

```diff
@@ -33,9 +33,9 @@ function SingleImageViewerContainer({
 }) {
   const [dragMove, setDragMove] = useState()
   // TODO: replace this with a better function to parse the image location from a subject.
-  const imageUrl = subject ? Object.values(subject.locations[frame])[0] : null
+  const imageLocation = subject ? subject.locations.find(l => l.type === 'image') : null
```

Comment:
Similar to comment in MultiFrameViewerContainer this is missing selecting the location per frame prop.

```suggestion
  const imageLocation = subject?.locations[frame].type === 'image' ? subject.locations[frame] : null
```

Reply:
```suggestion
  const imageLocation = subject ? subject.locations[frame] : null
```